### ISSUE TITLE
Add label to main FAB

### ISSUE THREAD

**Issue by leinardi** [leinardi/FloatingActionButtonSpeedDial#10]
Add a label also to the main FAB, so that can be used to replicate the exact same behavior of Inbox by Gmail and Calendar apps.

Since I would like to use the `FabWithLabelView`, this requires #11.

**Comment by leinardi** [comment]
Since I would like to use the `FabWithLabelView`, this is blocked by #11

**Comment by HowardSchmaeu** [comment]
YESS !!! That would be nice to have!!!

**Comment by IslamKhSh** [comment]
it's very important to have
